style.css

```diff
@@ -1,2 +1,215 @@
-body { font-family: "Montserrat", serif; }
+body {
+  background: #fff;
+  font-family: "Montserrat", serif;
+  height: 2700px;
+  width: 1440px;
+}
 
+.wordmarksmall {
+  height: 51px;
+  margin-bottom: 2625px;
+  margin-left: 43px;
+  margin-right: 258px;
+  margin-top: 24px;
+  position: absolute;
+  width: 166px;
+}
+
+.nav {
+  display: flex;
+  margin-left: 258px;
+  width: 1140px;
+}
+
+.nav__link { 
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6044074)